resource/plugins/unreal/python/publisher/collectors/unreal_dependencies_publisher_collector.py

```diff
@@ -0,0 +1,46 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+
+import ftrack_api
+
+import unreal
+
+from ftrack_connect_pipeline_unreal import plugin
+from ftrack_connect_pipeline_unreal.utils import (
+    custom_commands as unreal_utils,
+)
+
+
+class UnrealLevelPublisherCollectorPlugin(
+    plugin.UnrealPublisherCollectorPlugin
+):
+    '''Unreal level/assets publisher collector plugin'''
+
+    plugin_name = 'unreal_dependencies_publisher_collector'
+
+    def fetch(self, context_data=None, data=None, options=None):
+        '''Fetch all cameras from the scene'''
+
+        print("Data on the fetch call ---> {}".format(data))
+        # TODO: take the level from the previous collector plugin executed:
+        #  In order to do this, we have to modify the plugin execution to pass
+        #  the data of the previous plugin in pipeline.
```

Comment:
Is this comment still vallid?

Reply:
No, this is handled by the batch publisher now